PULL_REQUEST_TEMPLATE/feature-frontend.md

```diff
@@ -7,6 +7,8 @@
 - [ ] feature tested manually 👨‍🔬 🧪
 - [ ] supporting test cases added 🚀 💼
 - [ ] code is readable - added necessary doc comments 👨‍🏭 🗒️
+- [ ] translation keys added 🌐 🗝️
+- [ ] dpqa automation attributes added 🤖 🧪
```

Comment:
```suggestion
- [ ] dpqa automation attributes added 🤖 🛠️
```

Reply:
Updated in c50849140a410ed50027f1d64b3ce19f057cf1f1